libs/services/designer-client-services/src/lib/standard/operationmanifest.ts

```diff
@@ -1,5 +1,6 @@
 import type { IHttpClient } from '../httpClient';
```

Comment:
Strongly oppose changes in this file :) these are already sent by service in standard so we should even try to hardcode these manifests.

Reply:
I went ahead and reverted a lot of this file, and I separated the services into base/standard/consumption versions so everything should be separated and organized corrected now.  If you think I need to make more changes I will adjust as needed!